### PR TITLE
feat: backport pipeline from #2260 [6.8]

### PR DESCRIPTION
**Pull request by v1v**
backport pipeline from https://github.com/elastic/apm-server/issues/2260

### CODE REVIEW

File: script/jenkins/check-changelogs.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+docker run --rm \
```
Comment (kuisathaverat) [inline script/jenkins/check-changelogs.sh]: I'd provision the docker container in the Jenkinsfile to not force to have docker to run the thing

Comment (v1v) [reply script/jenkins/check-changelogs.sh]: Not sure though, I'd rather prefer the script to be simple and then able to run it locally

Comment (kuisathaverat) [reply script/jenkins/check-changelogs.sh]: I've created https://github.com/elastic/apm-server/issues/2442

**Comment by v1v** [comment]
After revisiting this PR I realize this is not required though as we the script itself should be branch independent as the list of branches are defined within the script. Therefore I'll close this PR now

**Pull request closed without merging**